OPAL/br/src/main/scala/org/opalj/br/analyses/cg/EntryPointFinder.scala

```diff
@@ -357,55 +359,37 @@ object AllEntryPointsFinder extends EntryPointFinder {
-        ("onDestroy", Some(MethodDescriptor.NoArgsAndReturnVoid))
-    )
-    val contentProviderEntryPoints: List[(String, Option[MethodDescriptor])] = List(
-        ("onCreate", Some(MethodDescriptor.NoArgsAndReturnVoid)),
-        ("query", None),
-        ("insert", None),
-        ("update", None)
-    )
-    val locationListenerEntryPoints: List[(String, Option[MethodDescriptor])] = List(
-        ("onLocationChanged", None),
-        ("onProviderDisabled", Some(MethodDescriptor.JustTakes(FieldType("Ljava/lang/String;")))),
-        ("onProviderEnabled", Some(MethodDescriptor.JustTakes(FieldType("Ljava/lang/String;")))),
-        ("onStatusChanged", Some(MethodDescriptor.apply(FieldTypes(FieldType("Ljava/lang/String;"), IntegerType, FieldType("Landroid/os/Bundle;")), VoidType)))
-    )
-    val broadcastReceiverEntryPoints: List[(String, Option[MethodDescriptor])] = List(
-        ("onReceive", Some(MethodDescriptor.apply(FieldTypes(FieldType("Landroid/content/Context;"), FieldType("Landroid/content/Intent;")), VoidType)))
-    )
-    val defaultEntryPoints = Map(
-        "android/app/Activity" -> activityEntryPoints,
-        "android/app/Service" -> serviceEntryPoints,
-        "android/content/ContentProvider" -> contentProviderEntryPoints,
-        "android/location/LocationListener" -> locationListenerEntryPoints,
-        "android/content/BroadcastReceiver" -> broadcastReceiverEntryPoints
-    )
-
     override def collectEntryPoints(project: SomeProject): Traversable[Method] = {
         val entryPoints = ArrayBuffer.empty[Method]
+
+        val defaultEntryPoints = Map(
+            "android/app/Activity" -> ListBuffer.empty[(String, Option[MethodDescriptor])],
+            "android/app/Service" -> ListBuffer.empty[(String, Option[MethodDescriptor])],
+            "android/content/ContentProvider" -> ListBuffer.empty[(String, Option[MethodDescriptor])],
+            "android/location/LocationListener" -> ListBuffer.empty[(String, Option[MethodDescriptor])],
+            "android/content/BroadcastReceiver" -> ListBuffer.empty[(String, Option[MethodDescriptor])]
+        )
+
+        val defEntry = project.config.getConfig("org.opalj.br.analyses.cg.DefaultEntryPoints")
+        defaultEntryPoints.keys.foreach { d ⇒
+            defEntry.getConfigList(d).forEach { entry ⇒
+                defaultEntryPoints(d) += ((entry.getString("name"), parseMethodDescriptor(entry.getString("descriptor"))))
+            }
+        }
+
         for ((superClass, methodList) ← defaultEntryPoints) {
             entryPoints ++= findEntryPoints(ObjectType(superClass), methodList, project)
         }
         entryPoints
     }
 
-    def findEntryPoints(ot: ObjectType, possibleEntryPoints: List[(String, Option[MethodDescriptor])], project: SomeProject): Set[Method] = {
+    def parseMethodDescriptor(descriptor: String): Option[MethodDescriptor] = {
+        if (descriptor != "None") {
```

Comment:
It might be better to, instead of explicitly putting the String "None" into the config, just leave it out. I think you should be able to get an optional value for the descriptor than that you can just `descriptor.map(MethodDescriptor(_))` then.